tests/components/simplefin/conftest.py

```diff
@@ -0,0 +1,73 @@
+"""Test fixtures for SimpleFIN."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import pytest
+from simplefin4py import FinancialData
+from simplefin4py.exceptions import SimpleFinInvalidClaimTokenError
+
+from homeassistant.components.simplefin import CONF_ACCESS_URL
+from homeassistant.components.simplefin.const import DOMAIN
+
+from tests.common import MockConfigEntry, load_json_value_fixture
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Mock setting up a config entry."""
+    with patch(
+        "homeassistant.components.simplefin.async_setup_entry", return_value=True
+    ) as mock_setup:
+        yield mock_setup
+
+
+@pytest.fixture
+def mock_access_url() -> str:
+    """Fixture to mock the access_url method of SimpleFin."""
+    return "https://i:am@yomama.house.com"
+
+
+@pytest.fixture
+async def mock_config_entry(mock_access_url) -> MockConfigEntry:
+    """Fixture for MockConfigEntry."""
+    return MockConfigEntry(
+        domain=DOMAIN,
+        data={CONF_ACCESS_URL: mock_access_url},
+        version=1,
+        unique_id=mock_access_url,
+    )
+
+
+@pytest.fixture
+def mock_get_financial_data() -> FinancialData:
+    """Fixture to mock the fetch_data method of SimpleFin."""
+    fixture_data = load_json_value_fixture("simplefin/fin_data.json")
+
+    fin_data = FinancialData(fixture_data)
+    with patch(
+        "homeassistant.components.simplefin.coordinator.SimpleFin.fetch_data",
```

Comment:
Rather mock the full lib object to avoid missing a patch in the future. You can create a shared mock for both the integration and config_flow, checkout `airgradient` for an example